Dockerfile

```diff
@@ -25,23 +25,23 @@ COPY . .
 
```

Comment:
This is as far up as Github will let me place a comment, but this is really about line 1: Let's bump the image to `swift:6.1-noble`.